drivers/mrf24j40/mrf24j40_netdev.c

```diff
@@ -330,6 +330,20 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
             }
             break;
 
+#ifdef MODULE_NETDEV_IEEE802154_OQPSK
+
+        case NETOPT_IEEE802154_PHY:
+            assert(max_len >= sizeof(int8_t));
+            *(uint8_t *)val = IEEE802154_PHY_OQPSK;
+            return sizeof(uint8_t);
```

Comment:
Why is this only enabled with `MODULE_NETDEV_IEEE802154_OQPSK`? Is there another option for the phy?

Reply:
I agree that this is a bit awkward.
I wanted a way to being able to disable it when it's not needed and as the `high_rate` option is only available with `netdev_ieee802154_oqpsk` I thought that's the way to go.

However this is explicitly a non-IEEE802.15.4 mode, so maybe having a separate `netdev_ieee802154_oqpsk_high_rate` pseudo-module would be better here.

Reply:
In this case `ifconfig` should be changed as well right?

Reply:
For conditional compilation, yes.

Reply:
I was actually referring to the `NETOPT_IEEE802154_PHY` option. Now that I understand what `NETOPT_OQPSK_RATE` is supposed to do, I think it's OK to enable this functionality just with `MODULE_NETDEV_IEEE802154_OQPSK`.